lib/calculate_format.rb

```diff
@@ -40,7 +40,11 @@ def any_of_format?(format)
     @cluster.ht_items.any? { |ht| item_format(ht) == format }
   end
 
-  def cluster_has_item_with_enum_and_same_ht_bib_key?(ht_item)
+  def record_has_any_enum?(ht_item)
+    # TODO: FIX with n_enum
```

Comment:
Can test this after https://github.com/hathitrust/holdings-backend/pull/330 is merged. I think the existing code there after the `return false` should work (although it might not be the most efficient)

Reply:
This seems to work and re-enables failing tests; however, (quoting the existing code below) the use of safe dereference on `ht.n_enum` leads to `nil` behaving opposite empty string. Is it expected that there will ever be a nil `n_enum` (and we would not want to bail out with an error)? My knee-jerk is to do something more like `!(ht.n_enum || "").empty?`
```
@cluster.ht_items.any? do |ht|
  ht.ht_bib_key == ht_item.ht_bib_key && !ht.n_enum&.empty?
end
```

Reply:
n_enum could be nil, and we should verify that it has the same behavior as "".

Reply:
I've re-enabled the `CalculateFormat` tests and added a test for the change. Reverting it fails the test.

Reply:
Removed the test as it was flawed and was not doing what I thought it was. Reverted to original form of `record_has_any_enum?` with a TODO comment.